.circleci/config.yml

```diff
@@ -63,3 +63,6 @@ workflows:
       - checks
       - tests
       - tests-embedapi
```

Comment:
```suggestion
      - tests-embedapi:
```

Reply:
Why is this needed?

Reply:
so it's a valid map, currently CI is failing bc of syntax error